client-react/src/pages/app/app-settings/SettingSourceColumn.tsx

```diff
@@ -28,26 +24,34 @@ const SettingSourceColumn: React.FC<SettingSourceColumnProps> = props => {
   const updatedValue = value?.toLowerCase();
   const filteredReference = references.filter(ref => ref.name.toLowerCase() === updatedName);
 
-  if (updatedValue?.startsWith(azureAppConfigRefStart)) {
-    return (
-      <div className={defaultCellStyle} aria-label={t('azureAppConfigValue')}>
-        {t('azureAppConfigRefValue')}
-      </div>
-    );
-  } else if (filteredReference.length > 0) {
-    return (
-      <div
-        className={defaultCellStyle}
-        aria-label={`${t('azureKeyVault')} ${!isKeyVaultReferenceResolved(filteredReference[0]) && 'not'} resolved`}>
-        <Icon
-          iconName={getKeyVaultReferenceStatusIconProps(filteredReference[0]).icon}
-          className={keyVaultIconStyle(theme, getKeyVaultReferenceStatusIconColor(filteredReference[0], theme))}
-          ariaLabel={t('azureKeyVault')}
-        />
-        <span className={sourceTextStyle}>{t('azureKeyVault')}</span>
-      </div>
-    );
-    // NOTE (krmitta): This value is shown only with the flag, and is currently for the private preview
+  if (filteredReference.length > 0) {
+    if (updatedValue?.startsWith(azureAppConfigRefStart)) {
+      return (
+        <div
+          className={defaultCellStyle}
+          aria-label={`${t('azureAppConfigValue')} ${!isReferenceResolved(filteredReference[0]) && 'not'} resolved`}>
+          <Icon
+            iconName={getReferenceStatusIconProps(filteredReference[0]).icon}
+            className={IconStyle(theme, getReferenceStatusIconColor(filteredReference[0], theme))}
```

Comment:
If `IconStyle` renamed to `iconStyle`:
```suggestion
            className={iconStyle(theme, getReferenceStatusIconColor(filteredReference[0], theme))}
```